packages/teams-js/test/resultValidation.ts

```diff
@@ -0,0 +1,52 @@
+import { MessageRequest } from './utils';
+
+export enum MatcherType {
+  ToBe,
+  ToStrictEqual,
+}
+
+// This function will return the number of assertions used to validate a request object using
+// validateRequestWithoutArguments or validateExpectedArgumentsInRequest.
+// The number of assertions used changes depending on how many arguments are being validated, so
+// you have to pass in the number of arguments being validated.
+// The value returned from this function can be used in calls to expect.assertions() if you have them.
+export function getNumberOfAssertionsUsedToValidateRequest(numberOfArgumentsBeingValidated: number): number {
+  return 2 + (numberOfArgumentsBeingValidated === 0 ? 0 : 2 + numberOfArgumentsBeingValidated);
```

Comment:
2?

Reply:
I can add a comment if you think that would make it clearer. It's literally the number of test assertions in these methods.

Reply:
See if it's any clearer now, I tried to replace some of the numbers with words -- but there's no way to automatically calculate these unfortunately.